ui/js/lbryuri.js

```diff
@@ -5,6 +5,8 @@ const lbryuri = {};
 
 lbryuri.REGEXP_INVALID_URI = /[^A-Za-z0-9-]/g;
 
+lbryuri.REGEXP_ADDRESS = /^b[HJK][1-9A-HJ-Za-km-z]{32}/;
```

Comment:
Kinda ugly, but you can't mix negative and positive character ranges. I tried to do it with lookaheads, since you can apply two of those to the same character, but ended up with a 40+ char monstrosity.

Reply:
i think it would need more then reg expression i could enter 32 a's in this case right?

Reply:
We decided to keep it simple and do the client-side address validation with a simple regex, and have the daemon return an error if the address isn't actually valid. So we're going to handle that case, but it's going to be a separate chunk of logic (probably in this same PR).

Reply:
Found a way to do it with lookahead that I guess is better? `/^b[HJK](?=[^0Il]{32})[0-9A-Za-z]{32}$/`

Reply:
I'm open to `trim` as a high-level prop on inputs that always trims spaces on the value.

Reply:
That makes sense. Although just to be clear, the lookahead regex is just meant to be more readable; it doesn't address the whitespace issue.